examples/particles.ipynb

```diff
@@ -274,7 +272,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "particle.load_particles(\"additional_particles.yml\")"
+    "from expertsystem.io import load_particle_collection\n",
+    "\n",
+    "additional_particles = load_particle_collection(\"additional_particles.yml\")\n",
+    "particle_db.merge(additional_particles)"
```

Comment:
You can also do:
```python
particle_db += load_particle_collection("additional_particles.yml")
```

Reply:
851205813fc1b65715ce96956de4eba3af04c9cb